example-get-started-cv/generate.sh

```diff
@@ -0,0 +1,197 @@
+dvc push
+
+
+cp -r $HERE/code/notebooks .
+pip install -r requirements.txt
+pip install jupyter
+jupyter nbconvert --execute 'notebooks/TrainSegModel.ipynb' --inplace
+git add .
+tick
+git commit -m "Add notebook using DVCLive"
+git tag -a "1-notebook-dvclive" -m "Notebook added."
+
+cp -r $HERE/code/src .
+cp $HERE/code/params.yaml .
+
+dvc stage add -n data_split \
+  -p base,data_split \
+  -d src/data_split.py -d data/pool_data \
+  -o data/train_data -o  data/test_data \
+  python src/data_split.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run data split stage"
+dvc push
+
+dvc stage add -n train \
+  -p base,train \
+  -d src/train.py -d data/train_data \
+  -o models/model.pkl \
+  -M results/train/metrics.json \
+  --plots-no-cache results/train/plots \
+  python src/train.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run train stage"
+dvc push
+
+dvc stage add -n evaluate \
+  -p base,evaluate \
+  -d src/evaluate.py -d models/model.pkl -d data/test_data \
+  -M results/evaluate/metrics.json \
+  --plots-no-cache results/evaluate/plots \
+  python src/evaluate.py
+dvc repro
+git add .
+tick
+git commit -m "Create and run evaluation stage"
+git tag -a "baseline-experiment" -m "Baseline experiment evaluation"
+dvc push
+
+export GIT_AUTHOR_NAME="David de la Iglesia"
+export GIT_AUTHOR_EMAIL="daviddelaiglesiacastro@gmail.com"
+export GIT_COMMITTER_NAME="$GIT_AUTHOR_NAME"
+export GIT_COMMITTER_EMAIL="$GIT_AUTHOR_EMAIL"
+
+git checkout -b "tune-architecture"
+
+unset GIT_AUTHOR_DATE
+unset GIT_COMMITTER_DATE
+
+dvc exp run --queue --set-param train.arch=alexnet
+dvc exp run --queue --set-param train.arch=resnet34
+dvc exp run --queue --set-param train.arch=vgg19_bn
+dvc exp run --queue --set-param train.arch=squeezenet1_1
+
+dvc exp run --run-all
+# Apply best experiment
+EXP=$(dvc exp show --csv --sort-by results/evaluate/metrics.json:dice_multi | tail -n 1 | cut -d , -f 1)
+dvc exp apply $EXP
+tick
+git commit -am "Run experiments tuning architecture"
+dvc push
+
+git checkout main
+
+popd
+
+unset TAG_TIME
+unset GIT_AUTHOR_DATE
+unset GIT_COMMITTER_DATE
+unset GIT_AUTHOR_NAME
+unset GIT_AUTHOR_EMAIL
+unset GIT_COMMITTER_NAME
+unset GIT_COMMITTER_EMAIL
+
+echo "`cat <<EOF-
+
+The Git repo generated by this script is intended to be published on
+https://github.com/iterative/example-get-started-cv. Make sure the Github repo
+exists first and that you have appropriate write permissions.
+
+Run these commands to force push it:
+
+cd build/example-get-started-cv
+git remote add origin git@github.com:<slug>/example-get-started-cv.git
+git push --force origin main
+git push --force origin tune-architecture
+git push --force origin increase-epochs
+git push --force origin --tags
+
+Run these to drop and then rewrite the experiment references on the repo:
+
+dvc exp remove -A -g origin
+dvc exp push origin -A
+
+To create a PR from the "tune-architecture" branch:
+
+gh pr create -t "Run experiments tuning architecture" \
```

Comment:
I and @alex000kim discussed that would be nice to create a P.R. via Studio but afaik this has to be done manually after the repo is generated

Reply:
yep, that's fine ... we do have a bit of manual stuff in the example-get-started repo